src/videojs-resumecard.scss

```diff
@@ -1,6 +1,6 @@
 /// @name VideoJS Resume Card
 
-:global(.vjs-resume-card)
+:global(.vjs-resumecard)
```

Comment:
Updated to be more consistent with other plugins, don't think it matters much either way.